build/orc-entrypoint.sh

```diff
@@ -5,7 +5,7 @@ set -o xtrace
 
 OPERATOR_BINDIR=/opt/percona
 ORC_CONF_PATH=${ORC_CONF_PATH:-/etc/orchestrator}
-ORC_CONF_FILE=${ORC_CONF_FILE:-"${ORC_CONF_PATH}/orchestrator.conf.json"}
+ORC_CONF_FILE=${ORC_CONF_FILE:-"${ORC_CONF_PATH}/mysql/orchestrator.conf.json"}
```

Comment:
why do we need this change?
and if we need it why do we use `mysql` as dir name?

Reply:
It is a good question! I will update description to explain it.

Reply:
I do not like mysql as well :(

Reply:
We agreed to rename config -> custom, mysql -> config.